libraries/AP_Proximity/AP_Proximity.cpp

```diff
@@ -308,17 +308,19 @@ AP_Proximity::Status AP_Proximity::get_instance_status(uint8_t instance) const
     return state[instance].status;
 }
 
+// Returns status of first good sensor. If no good sensor found, returns status of last instance sensor
 AP_Proximity::Status AP_Proximity::get_status() const
 {
+    Status sensors_status = Status::NotConnected;
```

Comment:
We don't really need this out here; could have a constant return value and a more-local variable introduced for `get_instance_status`

Reply:
I kept it that way because the status could be "not connected" or "no data".. I would like the sensor to return the value that is true for a sensor, rather than an arbitrary constant